keanu-project/src/main/java/io/improbable/keanu/network/ProtobufWriter.java

```diff
@@ -98,11 +109,13 @@ private void saveParents(KeanuSavedBayesNet.Vertex.Builder vertexBuilder,
         return parentBuilder.build();
     }
 
+    @Override
     public void saveValue(Vertex vertex) {
         //TODO - Remove once we have a version for all Vertex data types
```

Comment:
What's this pesky `TODO` doing here

Reply:
It's a temp thing that will get removed as part of a final pass of the APIs (once all of this + the dot stuff is out).  It needs to be there at the moment though.